### ISSUE TITLE
`--remove-container` has no effect

### ISSUE THREAD

**Issue by mwarres** [oxsecurity/megalinter#1917]
`--remove-container` has no effect

To Reproduce:
1. Run `docker system prune`.
2. Run `docker container list --all`, verifying that there are no containers present.
3. Run `npx -- mega-linter-runner@v6.10.0 --remove-container --flavor documentation --release v6.10.0`.
4. Run `docker container list --all`.

Expected Behavior:
Expected the final output to be empty.  

Actual Behavior:
Output of step 4. is:
```
CONTAINER ID   IMAGE                                         COMMAND                  CREATED         STATUS                          PORTS     NAMES
747b578665c6   oxsecurity/megalinter-documentation:v6.10.0   "/bin/bash /entrypoi…"   2 minutes ago   Exited (0) About a minute ago             keen_hamilton
```
Thank you for taking a look at this!!

**Comment by nvuillam** [comment]
@Kurt-von-Laven , any idea ? ( you're the local runner expert :p )

**Comment by Kurt-von-Laven** [comment]
As it turns out our community solved this issue only last week, so I opened a pull request with the suggested fix and filed gkz/optionator#42 to spare others similar embarrassment in the future.

**Comment by mwarres** [comment]
Thank you so much for fixing this!!
